frontend/app/tests/unit/specs/components/settings/onboarding-settings.spec.ts

```diff
@@ -0,0 +1,228 @@
+import { type Wrapper, mount } from '@vue/test-utils';
+import Vuetify from 'vuetify';
+import { type Pinia } from 'pinia';
+import { beforeEach } from 'vitest';
+import OnboardingSettings from '@/components/settings/OnboardingSettings.vue';
+import { useMainStore } from '@/store/main';
+
+vi.mock('@/composables/electron-interop', () => ({
+  useInterop: vi.fn().mockReturnValue({
+    isPackaged: true,
+    restartBackend: vi.fn(),
+    config: vi.fn().mockReturnValue({
+      logDirectory: '/Users/home/rotki/logs'
+    })
+  })
+}));
+
+let saveOptions = vi.fn();
+vi.mock('@/composables/backend', async () => {
+  const mod = await vi.importActual<typeof import('@/composables/backend')>(
+    '@/composables/backend'
+  );
+  return {
+    ...mod,
+    useBackendManagement: vi.fn().mockImplementation(loaded => {
+      const mocked = mod.useBackendManagement(loaded);
+      saveOptions = vi.fn().mockImplementation(opts => {
+        mocked.saveOptions(opts);
+      });
+      return {
+        ...mocked,
+        saveOptions
+      };
+    })
+  };
+});
+
+vi.mock('@/composables/api/info/index', () => ({
+  useInfoApi: vi.fn().mockReturnValue({
+    info: vi.fn().mockReturnValue({
+      version: {
+        our_version: '1.28.0',
+        latest_version: null,
+        download_url: null
+      },
+      dataDirectory: '/Users/home/rotki/develop_data',
+      logLevel: 'DEBUG',
+      acceptDockerRisk: false,
+      backendDefaultArguments: {
+        maxLogfilesNum: 3,
+        maxSizeInMbAllLogs: 300,
+        sqliteInstructions: 5000
+      }
+    }),
+    ping: vi.fn().mockReturnValue(true)
+  })
+}));
+
+vi.mock('@/composables/api/settings/settings-api', () => ({
+  useSettingsApi: vi.fn().mockReturnValue({
+    backendSettings: vi.fn().mockReturnValue({
+      maxSizeInMbAllLogs: {
+        value: 300,
+        isDefault: true
+      },
+      maxLogfilesNum: {
+        value: 3,
+        isDefault: true
+      },
+      sqliteInstructions: {
+        value: 5000,
+        isDefault: true
+      }
+    })
+  })
+}));
+
+describe('OnboardingSetting.vue', () => {
+  let pinia: Pinia;
+  let wrapper: Wrapper<OnboardingSettings>;
+
+  function createWrapper() {
+    const vuetify = new Vuetify();
+    return mount(OnboardingSettings, {
+      pinia,
+      vuetify
+    });
+  }
+
+  beforeAll(async () => {
+    const pinia = createPinia();
+    setActivePinia(pinia);
+    await useMainStore().getInfo();
+  });
+
+  beforeEach(async () => {
+    wrapper = createWrapper();
+    await wrapper.vm.$nextTick();
+  });
+
+  describe('standard settings', () => {
+    test('use default value from info api or electron config, save button should be disabled', async () => {
```

Comment:
After merging on develop we should change this too to follow the new approach in form validation.